docs_edit_panel/main.js

```diff
@@ -21,7 +21,8 @@ const app = createApp({
     updateColumns(columns) {
       this.docs.columns = columns.map((column) => {
         const existingColumn = this.docs?.columns.find(
-          (existingColumn) => column.name === existingColumn.name,
+          (existingColumn) =>
+            column.name.toLowerCase() === existingColumn.name.toLowerCase(),
```

Comment:
I think this should obey quoted identifier settings of dbt to work consistently.

For example A != "a" in snowflake.